src/components/Homepage.js

```diff
@@ -1,23 +1,12 @@
-import { useState } from 'react';
+// import { useState } from 'react';
```

Comment:
- [OPTIONAL] Kindly make sure to remove all comments in your codebase in order to keep it clean and professional.